README.md

```diff
@@ -44,27 +44,34 @@ npm config set externalplugins ../relative/path/to/plugins/directory
 
 ## Adding npm Package Dependencies
 
-If you want to add a new npm package to 'node_modules':
+If you want to add a new npm package to 'node_modules' you will need to `--save-exact`:
 
 1. Install the new package
-  * Install and save dependencies in `package.json`
+  * The command below will install and save dependencies in `package.json`
 
     ```
-    npm install [your package] --save
+    npm install [your package] --save --save-exact
     ```
 
-  * Then, add the package to devDependencies
+  * Then, (if needed) add the package to devDependencies
 
     ```
-    npm install [your package] --save-dev
+    npm install [your package] --save-dev --save-exact
     ```
 
 2. Create a synced npm-shrinkwrap.json with devDependencies included by running
 
     ```
+    npm shrinkwrap
+    ```
+and/or
+
+    ```
     npm shrinkwrap --dev
     ```
 
+ps: after generating the `shrinkwrap` you will need to remove the `fsevents` from the updates since it's automatically added and not needed.
```

Comment:
Also it would be nice to give some more context why `fsevents` is removed. 

```
// remove 'fsevents' from shrinkwrap, since it causes errors on non-Mac hosts
// see https://github.com/npm/npm/issues/2679
```
from [./gulpfile.js#L88](https://github.com/dcos/dcos-ui/blob/master/gulpfile.js#L88)